doc/Relationships.md

```diff
@@ -223,6 +223,11 @@ When displaying a post:
 
 ![embedded_list in editionView](images/embedded_list_in_editionView.png)
 
+Caution: If you want to use [`Field.template()`](./Theming.md#customizing-the-template-for-a-given-field)
```

Comment:
i'd be happy to, but I'm not quite clear where you want this explanation. or what text you want in addition to what I've already put, sorry. Perhaps you should add an explanation that makes sense to you directly to the code? Or let me know in more detail what changes you had in mind, and I'll make them.

(Also perhaps this issue about template() would be better addressed on its own Issue thread, as it's separate from the problem in #1158)